pkg/cmd/pr/shared/finder.go

```
@@ -59,7 +59,8 @@ func NewFinder(factory *cmdutil.Factory) PRFinder {
 		httpClient: factory.HttpClient,
 		progress:   factory.IOStreams,
 		branchConfig: func(s string) git.BranchConfig {
-			return factory.GitClient.ReadBranchConfig(context.Background(), s)
+			branchConfig, _ := factory.GitClient.ReadBranchConfig(context.Background(), s)
```

Should this panic if an error is returned?

---

🤔 That's a great question... It shouldn't, and it should probably return the error now that it's good and properly handled...

---

Sooooooo this looks like a can of worms 😅 I've done the absolute minimum to surface this error in the finder for now, but at first blush [parseCurrentBranch in shared/finder.go](https://github.com/cli/cli/blob/21a1f8e8a9201143b11cb8b8e5d78bc163839c60/pkg/cmd/pr/shared/finder.go#L235-L283) looks awfully similar [prSelectorForCurrentBranch pr/status/status.go](https://github.com/cli/cli/blob/21a1f8e8a9201143b11cb8b8e5d78bc163839c60/pkg/cmd/pr/status/status.go#L187-L222)...

I'm not gonna investigate that similarity further at this time